string obj1
string obj1_type
string obj2
string obj2_type
string spatial_relation
int32 start_frame
int32 end_frame
